repo/packages/C/chronos/0/marathon.json

```diff
@@ -28,6 +28,8 @@
       "portIndex": 0,
       "protocol": "HTTP"
     }
-  ]
-
+  ],
+  "labels": {
+    "DCOS_PACKAGE_FRAMEWORK_NAME": "{{chronos/mesos-framework-name}}"
```

Comment:
FYI, Ideally we shouldn't expose this label stuff in the Marathon JSON. It just happen that right now we are using Marathon to store this information. This is a short term solution and very implementation specific. We should try to mimic how the Marathon package does it by using hierarchical config.json templates and a well known name.